canonical-websites/www.ubuntu.com
# VMWare-to-openstack landing page v0

## Done

A first landing page example hosted on ubuntu.com

- [Copydoc](https://docs.google.com/document/d/1Wt8fkmkTbWJowrujNwsMna8SwHr24VUnPB0GM7sxGco/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/engage](http://0.0.0.0:8001/engage)
- Ensure it redirects to `/resources`
- View the site locally in your web browser at: [http://0.0.0.0:8001/engage/vmware-to-openstack](http://0.0.0.0:8001/engage/vmware-to-openstack)
- Ensure the landing page is displayed correctly (signed in and out of Brighttalk)

## Screenshots

![ef50bc31-cc9d-4511-b61c-880f7a2f653b](https://user-images.githubusercontent.com/18480003/40363456-dbadee8e-5dcf-11e8-87d4-fd1725de7cbb.png)
